### ISSUE TITLE
plugin focus in darktable.css

### ISSUE THREAD

**Issue by phweyland** [darktable-org/darktable#3093]
in darktable.css we have

```
#lib-plugin-ui *:hover,
#iop-plugin-ui *:hover,
#lib-plugin-ui *:focus,
#lib-plugin-ui *:focus
{
  font-weight: 600;
}
```

- the two focus lines are duplicated. one should be iop-plugin-ui.
- the effect for lib-plugin-ui is a bit weird IMHO. I suspect it would be the same for iop.

Without focus:
![image](https://user-images.githubusercontent.com/23012047/66397919-2176f880-e9dd-11e9-8cb8-100770f7abcf.png)

With focus:
![image](https://user-images.githubusercontent.com/23012047/66397971-394e7c80-e9dd-11e9-89e1-7616c090ff6d.png)

**Comment by phweyland** [comment]
The hover part has no effect (at least on tree-view).

What was the purpose of this setting ?

**Comment by hellbuster** [comment]
I suspect that the intent was to make the selected entry bold, I also found the lines suspicious, but I didn't found how to implement what I believe is the desired behavior ☹

**Comment by phweyland** [comment]
> I suspect that the intent was to make the selected entry bold,

May make sense for a single entry (even not sure) but highlighting the full list/tree doesn't make sense.
I think we should remove that. If the highlight makes sense for a specific item the setting should be attached to that item only and not all the sub elements of the panels.

**Comment by Nilvus** [comment]
@phweyland : It seems your issue is no more available since last PR by @aurelienpierre about UI fixes and mine #3362 where I totally removed these finally unused lines. And now hover are normally correct.
If it's now ok for you, could you close this issue ?

**Comment by phweyland** [comment]
Exact ! I close it. Thanks.
